<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1045</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QComboBox" name="vexNum">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>130</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
    border: 1px solid gray;   /* 边框 */
    border-radius: 4px;   /* 圆角 */
    color: #000;
    font: normal normal 15px &quot;Microsoft YaHei&quot;;
    background: transparent;
}</string>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>11</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>13</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>14</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>15</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>16</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>17</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>18</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>19</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="st">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>250</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
    border: 1px solid gray;   /* 边框 */
    border-radius: 4px;   /* 圆角 */
    /*padding: 1px 18px 1px 3px;    字体填衬 */
    color: #000;
    font: normal normal 15px &quot;Microsoft YaHei&quot;;
    background: transparent;
}

</string>
   </property>
   <item>
    <property name="text">
     <string>0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>11</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>13</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>14</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>15</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>16</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>17</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>18</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>19</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="en">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>250</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
    border: 1px solid gray;   /* 边框 */
    border-radius: 4px;   /* 圆角 */
    padding: 1px 18px 1px 3px;   /* 字体填衬 */
    color: #000;
    font: normal normal 15px &quot;Microsoft YaHei&quot;;
    background: transparent;
}

</string>
   </property>
   <item>
    <property name="text">
     <string>0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>11</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>13</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>14</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>15</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>16</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>17</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>18</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>19</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="w">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>190</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/*QSpinBox*/
QSpinBox {
padding-right: 10px; /* make room for the arrows */
border:1px solid black;
border-radius:5px;
background: transparent;
font: normal normal 15px &quot;Microsoft YaHei&quot;;

}

QSpinBox::up-button:pressed {
margin-top:3px;
}

QSpinBox::down-button:pressed {
margin-bottom:3px;
}
</string>
   </property>
  </widget>
  <widget class="QComboBox" name="arcNum">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>130</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
    border: 1px solid gray;   /* 边框 */
    border-radius: 4px;   /* 圆角 */
    padding: 1px 18px 1px 3px;   /* 字体填衬 */
    color: black;
    font: normal normal 15px &quot;Microsoft YaHei&quot;;
    background: transparent;
}</string>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>11</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>13</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>14</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>15</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>16</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>17</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>18</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>19</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>20</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>21</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>22</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>23</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>24</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>25</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>26</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>27</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>28</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>29</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>30</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="ensure2">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>230</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
            background-color:#ffd6a5;
            
            border-width:4px;
            border-radius:25px;
            border-color:rgba(230,230,250,30);
            font:bold 20px;
            color:black;
            padding:6px;
            }
QPushButton:pressed{
            background-color:#fdffb6;
            border-color:rgba(230,230,250,30);
            border-style:inset;
            color:black;
            }
 QPushButton:hover{
            background-color:#fdffb6;
			border-width:4px;
            border-color:#fffffc;
            color:black;
            }</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="adj">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>340</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
            background-color:#ffd6a5;
            border-width:4px;
 			border-style:outset;
            border-radius:6px;
            border-color:rgba(230,230,250,30);
            font:bold 20px;
            color:black;
            padding:6px;
            }
QPushButton:pressed{
            background-color:#fdffb6;
            border-color:#a0c4ff;
            border-style:inset;
            color:black;
            }
 QPushButton:hover{
            background-color:#fdffb6;
            border-color:#a0c4ff;
            color:black;
            }</string>
   </property>
   <property name="text">
    <string>显示邻接链表</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tupo">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>340</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
            background-color:#ffd6a5;
            border-style:outset;
            border-width:4px;
            border-radius:6px;
            border-color:rgba(230,230,250,30);
            font:bold 20px;
            color:black;
            padding:6px;
            }
QPushButton:pressed{
            background-color:#fdffb6;
            border-color:#a0c4ff;
            border-style:inset;
            color:black;
            }
 QPushButton:hover{
            background-color:#fdffb6;
            border-color:#a0c4ff;
            color:black;
            }</string>
   </property>
   <property name="text">
    <string>显示拓扑结构</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cri">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>440</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
            background-color:#ffd6a5;
            border-style:outset;
            border-width:4px;
            border-radius:6px;
            border-color:rgba(230,230,250,30);
            font:bold 20px;
            color:black;
            padding:6px;
            }
QPushButton:pressed{
            background-color:#fdffb6;
            border-color:#a0c4ff;
            border-style:inset;
            color:black;
            }
 QPushButton:hover{
            background-color:#fdffb6;
            border-color:#a0c4ff;
            color:black;
            }</string>
   </property>
   <property name="text">
    <string>显示关键路径</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ensure1">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>120</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
            background-color:#ffd6a5;
            
            border-width:4px;
            border-radius:25px;
            border-color:rgba(230,230,250,30);
            font:bold 20px;
            color:black;
            padding:6px;
            }
QPushButton:pressed{
            background-color:#fdffb6;
            border-color:rgba(230,230,250,30);
            border-style:inset;
            color:black;
            }
 QPushButton:hover{
            background-color:#fdffb6;
			border-width:4px;
            border-color:#fffffc;
            color:black;
            }</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="example">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>40</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
            background-color:#ffd6a5;
            
            border-width:1px;
            border-radius:4px;
            border-color:rgba(230,230,250,30);
            font:bold 20px;
            color:black;
            padding:6px;
            }
QPushButton:pressed{
            background-color:#fdffb6;
            border-color:rgba(230,230,250,30);
            border-style:inset;
            color:black;
            }
 QPushButton:hover{
            background-color:#fdffb6;
			border-width:4px;
            border-color:#fffffc;
            color:black;
            }</string>
   </property>
   <property name="text">
    <string>使用示例</string>
   </property>
  </widget>
  <widget class="QPushButton" name="look">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>440</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
            background-color:#ffd6a5;
            border-style:outset;
            border-width:4px;
            border-radius:6px;
            border-color:rgba(230,230,250,30);
            font:bold 20px;
            color:black;
            padding:6px;
            }
QPushButton:pressed{
            background-color:#fdffb6;
            border-color:#a0c4ff;
            border-style:inset;
            color:black;
            }
 QPushButton:hover{
            background-color:#fdffb6;
            border-color:#a0c4ff;
            color:black;
            }</string>
   </property>
   <property name="text">
    <string>关键路径详情</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>140</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: &quot;Microsoft YaHei&quot;;
    font-size: 18px;
	border-radius:5px;
    color: black;
    background-color: #bdb2ff;
}
</string>
   </property>
   <property name="text">
    <string>点数</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>140</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: &quot;Microsoft YaHei&quot;;
    font-size: 18px;
	border-radius:5px;
    color: black;
    background-color: #bdb2ff;
}
</string>
   </property>
   <property name="text">
    <string>边数</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>50</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: &quot;Microsoft YaHei&quot;;
    font-size: 18px;
	border-radius:5px;
    color: black;
    background-color: #bdb2ff;
}
</string>
   </property>
   <property name="text">
    <string>请在下方填选建图信息或</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>200</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: &quot;Microsoft YaHei&quot;;
    font-size: 18px;
	border-radius:5px;
    color: black;
    background-color: #bdb2ff;
}
</string>
   </property>
   <property name="text">
    <string>请依次建立边</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>260</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: &quot;Microsoft YaHei&quot;;
    font-size: 18px;
	border-radius:5px;
    color: black;
    background-color: #bdb2ff;
}
</string>
   </property>
   <property name="text">
    <string>起点</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>260</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: &quot;Microsoft YaHei&quot;;
    font-size: 18px;
	border-radius:5px;
    color: black;
    background-color: #bdb2ff;
}
</string>
   </property>
   <property name="text">
    <string>终点</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>200</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: &quot;Microsoft YaHei&quot;;
    font-size: 18px;
	border-radius:5px;
    color: black;
    background-color: #bdb2ff;
}
</string>
   </property>
   <property name="text">
    <string>边权</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reset">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>560</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
            background-color:#ffadad;
            border-style:outset;
            border-width:4px;
            border-radius:5px;
            border-color:rgba(230,230,250,30);
            font:bold 20px;
            color:black;
            padding:6px;
            }
QPushButton:pressed{
            background-color:#ffd6a5;
            border-color:rgba(230,230,250,30);
            border-style:inset;
            color:black;
            }
 QPushButton:hover{
            background-color:#ffd6a5;
            border-color:rgba(230,230,250,200);
            color:black;
            }</string>
   </property>
   <property name="text">
    <string>重  置</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
